stable/aws-calico/templates/rbac.yaml

```diff
@@ -0,0 +1,181 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: calico-node
```

Comment:
All object names should be `{{ template "aws-calico.fullname" . }}` see https://github.com/aws/eks-charts/blob/master/stable/appmesh-controller/templates/rbac.yaml#L5

Reply:
done